packages/grafana-ui/src/components/Gauge/Gauge.test.tsx

```diff
@@ -135,6 +135,45 @@ describe('Format value with value mappings', () => {
     expect(result.text).toEqual('1-20');
   });
 
+  it('should return if value is null and value to text mapping value is null', () => {
+    const valueMappings: ValueMapping[] = [
+      { id: 0, operator: '', text: '1-20', type: MappingType.RangeToText, from: '1', to: '20' },
+      { id: 1, operator: '', text: '<NULL>', type: MappingType.ValueToText, value: 'null' },
+    ];
+    const value = null;
+    const { instance } = setup({ valueMappings });
+
+    const result = instance.getFirstFormattedValueMapping(valueMappings, value);
+
+    expect(result.text).toEqual('<NULL>');
+  });
+
+  it('should return if value is null and range to text mapping from is null', () => {
+    const valueMappings: ValueMapping[] = [
+      { id: 0, operator: '', text: '<NULL>', type: MappingType.RangeToText, from: 'null', to: '10' },
```

Comment:
not sure range mapping should support null, what does null to 10 mean?

Reply:
Yes, when I create a new Singlestat I get a default ValueMapping with null => N/A. If I switch to Ranged mapping instead the default ValueMapping is switched to from: null to: null => N/A so I thought we need that for ranged value mappings too. Maybe the test/code should be for from and to equal to null?

Reply:
Will look into the getMappedValue(valueMappings: ValueMapping[], value: TimeSeriesValue) solution also :)

Reply:
yea, I think for range mapping to work with null, it needs to be from: null to: null, (still pretty confusing). Probably a bug in the singlestat value mapping that i defaults to that for range maps. But it does mean that singlestat thresholds probably have that out there in the wild.

Reply:
There might be some out there in the wild yes!